apps/directory/src/functions/viewmodelMapper.js

```diff
@@ -367,22 +367,26 @@ export const getNameOfHead = (head) => {
 };
 
 export const mapContactInfo = (instance) => {
-  return {
-    name: {
-      value: getNameOfHead(instance.head),
-      type: "string",
-    },
-    website: { value: mapUrl(instance.url), type: "url" },
-    email: {
-      value: instance.contact ? instance.contact.email : undefined,
-      type: "email",
-    },
-    juridical_person: { value: instance.juridical_person, type: "string" },
-    country: {
-      value: instance.country
-        ? instance.country.label || instance.country.name
-        : undefined,
-      type: "string",
-    },
-  };
+  if (instance.contact) {
```

Comment:
why do we like this view model exactly?

Reply:
feels like a duplication

Reply:
this is an old bit of code that just has been ported. when it all works it might be a good thing to take some time to make it better. This is the 2nd migration of the app (and a very big one)